src/components/Header/ThemeToggleButton.css

```diff
@@ -27,6 +27,24 @@
 	opacity: 1;
 }
 
+@media (max-width: 50em) {
+	.theme-toggle {
+		margin-top: 32px;
+	}
+	.hide-toggle-on-smaller-screens {
+		display: none;
+	}
+}
+
+@media (min-width: 50em) {
+	.theme-toggle {
+		display: none;
+	}
+	.hide-toggle-on-smaller-screens {
+		display: inline-flex;
+	}
+}
```

Comment:
We have an edge case at 800px wide where the margin-top persists and the toggle is already in the top bar:

<img width="784" alt="image" src="https://user-images.githubusercontent.com/357379/172683503-3d9d85b2-de36-4900-9cd3-9d301754da3a.png">

I think we can avoid the `max-width` query and just progressively enhance from mobile first:

```suggestion
.theme-toggle {
	margin-top: 32px;
}
.hide-toggle-on-smaller-screens {
	display: none;
}


@media (min-width: 50em) {
	.theme-toggle {
		display: none;
		margin-top: 0;
	}
	.hide-toggle-on-smaller-screens {
		display: inline-flex;
	}
}
```

(The `margin-top` should probably be in the main `.theme-toggle` declaration above but I can’t do that in a suggestion here.)